package.json

```diff
@@ -4,11 +4,15 @@
   "description": "Build/Layout optimization tooling and Peregrine framework adapters for the Magento PWA",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist --ignore __tests__ --source-maps --copy-files",
-    "lint": "eslint '*.js' 'src/**/*.js'",
     "test": "run-p jest:ci prettier:check lint",
-    "test:dev": "jest --watch",
-    "jest:ci": "jest -i --forceExit --testResultsProcessor=./node_modules/jest-junit-reporter",
+    "test:dev": "run-s prettier lint jest:ci",
+    "test:watch": "jest --watch --runInBand",
+    "test:debug": "node --inspect-brk ./node_modules/.bin/jest --runInBand",
+    "test:watch:focus": "jest --watch --runInBand --coverageReporters=text --collectCoverageFrom=\"${FOCUS}/**/*.js\" --no-cache \"${FOCUS}.*\"",
```

Comment:
Similar comment to another repo - don't want to overwhelm contributors with a billion options. This would be a good candidate for a local config